index.html

```diff
@@ -6,7 +6,37 @@
 <p>18F Guides is the repository for best practices across our teams. 
 Our guides are open source, and you're free to use them as you wish.
 Our hope is that other digital service teams — both inside and outside the government — 
-will adopt or modify the practices outlined here. </p> 
+will adopt or modify the practices outlined here.</p>
+
+<p>Here's a curated list of guides found both here and elswhere that will help you get started:</p>
+
+##Management
```

Comment:
Yeah, I didn't know if it was a way you'd want to ultimately structure the guides or not, but it seems like grouping them somewhat might be helpful for the user.  If that is preferable I'd be happy to close this request and submit a new one w/ child pages, but Emileigh / Andrew might have a preferred way of organizing the content.

Also noticed that the guides aren't consistently named - some are "18F Guide for XX", others are just a doc name, might be nice to standardize?

Thanks!